tools/plugin/alsaplug/tplg.c

```diff
@@ -101,20 +101,29 @@ static int plug_aif_in_out(snd_sof_plug_t *plug, int dir)
 	if (ret < 0)
 		return ret;
 
-	comp_info->ipc_payload =  calloc(sizeof(struct ipc4_base_module_cfg), 1);
+	comp_info->ipc_size = sizeof(struct ipc4_base_module_cfg) + sizeof(struct sof_uuid);
+	comp_info->ipc_payload =  calloc(comp_info->ipc_size, 1);
 	if (!comp_info->ipc_payload)
 		return -ENOMEM;
 
-	comp_info->ipc_size = sizeof(struct ipc4_base_module_cfg);
-
 	if (dir == SOF_IPC_STREAM_PLAYBACK) {
+		struct sof_uuid uuid = {.a = 0xe2b6031c, .b = 0x47e8, .c = 0x11ed,
+					.d = { 0x07, 0xa9, 0x7f, 0x80, 0x1b, 0x6e, 0xfa, 0x6c }};
 		comp_info->module_id = 0x96;
 		plug_setup_widget_ipc_msg(comp_info);
+		comp_info->uuid = uuid;
 	} else {
+		struct sof_uuid uuid = {.a = 0xdabe8814, .b = 0x47e8, .c = 0x11ed,
+					.d = { 0xa5, 0x8b, 0xb3, 0x09, 0x97, 0x4f, 0xec, 0xce }};
```

Comment:
can we have at least comments for these two? Are they only used here?

Reply:
ack, IIRC these override the endpoints with file module - would be good to spell out

Reply:
ack. fixed now for both the DAI and AIF compoenents